quickstart/text.xml

```diff
@@ -74,7 +74,7 @@ At the top of the ebuild is a header block. This is present in all ebuilds.
 </p>
 
 <p>
-Ebuilds are indented using tabs, with each tab representing four places.
+Ebuilds are indented using tabs, with each tab representing four spaces.
```

Comment:
That's not necessarily correct. A tab may represent less than 4 spaces if it doesn't start at a column that is divisible by 4. How about the following wording?
```suggestion
Ebuilds are indented using tabs, with a tab stop every four positions.
```

Reply:
I was using [ebuild-writing/file-format](https://devmanual.gentoo.org/ebuild-writing/file-format/index.html#indenting-and-whitespace) as reference, where it says spaces

Reply:
Hm, I guess for indenting it is correct.